### PR TITLE
Add Heroku deploy button

### PR DESCRIPTION
**Pull request by matthewford**
In order to deploy without modifying the config in the repo I added a number of
additional environment variables:

    SLACK_CHANNEL
    GITHUB_REPOS
    GITHUB_MEMBERS
    GITHUB_USE_LABELS
    GITHUB_EXCLUDE_LABELS
    GITHUB_EXCLUDE_TITLES

This only works in simple team structures (one channel for the whole team)

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "repository": "https://github.com/binaryberry/seal",
+  "env": {
+    "SEAL_ORGANISATION": {
+      "description": "Your Github organisation name"
+    },
+    "GITHUB_TOKEN": {
+      "description": "Your Github token (obtainable at https://github.com/settings/tokens)"
+    },
+    "GITHUB_REPOS": {
+      "description": "Comma serperated list of repos to check"
```
Comment (johnsyweb) [inline app.json]: "separated"

File: lib/seal.rb [hunk 2]
```diff
@@ -33,26 +33,41 @@ def teams
...
     message = message_builder.build
-    slack = SlackPoster.new(ENV['SLACK_WEBHOOK'], team_config(team)['channel'], message_builder.poster_mood)
+    channel = ENV["SLACK_CHANNEL"] ? ENV["SLACK_CHANNEL"] : team_config(team)['channel']
+    slack = SlackPoster.new(ENV['SLACK_WEBHOOK'], channel, message_builder.poster_mood)
     slack.send_request(message)
   end
 
   def org_config
-    @org_config ||= YAML.load_file("./config/#{ORGANISATION}.yml")
+    @org_config ||= YAML.load_file('./config/#{ORGANISATION}.yml') if File.exist?('./config/#{ORGANISATION}.yml')
```
Comment (johnsyweb) [inline lib/seal.rb]: The number of conditionals in this class concern me.

It feels like we need `EnvironmentConfiguration` and `FileConfiguration` `class`es with a common `Configuration` interface.

**Comment by johnsyweb** [comment]
I :heart: this idea but have concerns about the implmentation.

File: lib/seal.rb (hunk 2 above)
Comment (johnsyweb) [inline lib/seal.rb]: Changing the quotations in `YAML.load_file('./config/#{ORGANISATION}.yml')` breaks the loading.

**Comment by matthewford** [comment]
@johnsyweb I agree the config implementation could be improved. I don't have time to look at this at the moment, but happy for someone else to take it over.

**Comment by binaryberry** [comment]
Thanks both, didn't know about the Heroku deploy button!

**Pull request merged**